/**:
    ros__parameters:
      bbox_x_min: -1.0
      bbox_x_max: 1.0
      bbox_y_min: -1.0
      bbox_y_max: 1.0
      bbox_z_min: -1.0
      bbox_z_max: 1.0
      map_path: "src/lidar-localization-ros2/map/levine.pcd"
      point_cloud_frame: livox_frame
      pose_topic: pcl_pose
      output_cloud_topic: true